routes/web.php

```diff
@@ -139,10 +136,10 @@
 });
 
 Route::middleware(['auth','accountant'])->prefix('accounts')->name('accounts.')->group(function (){
-  Route::get('sectors','AccountController@sectors');
-  Route::post('create-sector','AccountController@storeSector');
-  Route::get('edit-sector/{id}','AccountController@editSector');
-  Route::post('update-sector','AccountController@updateSector');
+  Route::get('sectors','AccountController@sectors')->name('sectors.index');
+  Route::post('create-sector','AccountController@storeSector')->name('sectors.create');
+  Route::get('edit-sector/{sector}','AccountController@editSector')->name('sectors.edit');
+  Route::patch('update-sector/{sector}','AccountController@updateSector')->name('sectors.update');
```

Comment:
You have to use `@method('PATCH')` in the update sector Form in order to `Route::patch` work.

Please test the changes you made in the PR in a browser. It would be very helpful for me to review code quickly and confidently.

Thanks for your time and effort.

Reply:
ok thanks.